lib/stripe_mock/data.rb

```diff
@@ -374,15 +374,20 @@ def self.mock_subscription(params={})
           data: [{
             id: 'si_1AwFf62eZvKYlo2C9u6Dhf9',
             created: 1504035973,
-            metadata: {},
+            metadata: {
+              # Added for avoiding nil exception
+              company_deleted: false
+            },
             object: 'subscription_item',
             plan: {
               amount: 999,
               created: 1504035972,
               currency: StripeMock.default_currency
             },
             quantity: 1
-          }]
+          }],
+          # Added missing attribute - it does not have another page of items
+          has_more: false
```

Comment:
This feels like it should probably be added to the upstream, no?

Reply:
this is a part of Stripe. Yes, it can be added

Reply:
Added in #904.